src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
```

Comment:
Consider adding a brief comment above the 'adder' function to explain its behavior. While the code is clear, a short description can be helpful for maintainability, especially when working in teams or returning to the code after a long time.